### ISSUE TITLE
SevTech: Ages crash on startup [MacOS ARM]

### ISSUE THREAD

**Issue by FightMan01** [PrismLauncher/PrismLauncher#489]
### Operating System

macOS

### Version of Prism Launcher

Prism Launcher 5.2

### Version of Qt

Qt 6.3.0

### Description of bug

SevTech: Ages modpack crash on startup

### Steps to reproduce

Download the latest SevTech: Ages and start the modpack

### Suspected cause

`java.lang.NoClassDefFoundError: com/mojang/text2speech/NarratorOSX`

https://mclo.gs/9rJlyYJ

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by DioEgizio** [comment]
Did you migrate from polymc/manymc?

**Comment by FightMan01** [comment]
Yes, from ManyMC, I followed this: https://github.com/PrismLauncher/PrismLauncher/issues/162

But Regrowth or AOF 5 works fine. Just SevTech crashes.

**Comment by DioEgizio** [comment]
not a launcher problem and old, closing
